### PR TITLE
i18n(es): Updated `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `configuration-reference.mdx`
* [`54ca70a`](https://github.com/withastro/docs/commit/54ca70a6ce1b99dc46875931d6de3b3cf6451696)

### CODE REVIEW

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -206,9 +206,11 @@ En el ejemplo de abajo, `astro dev` empezará tu servidor en `/docs`.
 
 Cuando utilices esta opción, todas tus importaciones de activos estáticos y URL deben agregar la base como prefijo. Puedes acceder a este valor a través de `import.meta.env.BASE_URL`.
 
+Por defecto, el valor de `import.meta.env.BASE_URL` incluye una barra diagonal al final. Si tienes la opción [`trailingSlash`](/en/reference/configuration-reference/#trailingslash) establecida en `'never'`, deberás agregarla manualmente en tus importaciones de activos estáticos y URLs.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:209]:
```suggestion
Por defecto, el valor de `import.meta.env.BASE_URL` incluye una barra diagonal al final. Si tienes la opción [`trailingSlash`](/es/reference/configuration-reference/#trailingslash) establecida en `'never'`, deberás agregarla manualmente en tus importaciones de activos estáticos y URLs.
```

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by sasoria — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**